cardano_node_tests/tests/test_dbsync.py

```diff
@@ -302,7 +302,7 @@ def test_epoch(self, cluster: clusterlib.ClusterLib):
         common.get_test_id(cluster)
 
         current_epoch = cluster.g_query.get_epoch()
-        epoch = current_epoch - 20 if current_epoch >= 20 else current_epoch
+        epoch = current_epoch - 1 if current_epoch >= 1 else current_epoch
```

Comment:
It is very likely we'll hit the race condition in epoch 0, so let's just skip the test when `current_epoch` is 0.

In normal circumstances the test will never run in epoch 0. This can happen only when using "fast" cluster instance, which is not the case for dbsync nightly. Therefore the skip can happen only in special circumstances, like running on dev cluster.